integration_test/iptables/iptablestest-lab.yaml

```diff
@@ -39,6 +39,71 @@ spec:
 ---
 apiVersion: v1
 kind: Pod
+metadata:
+  name: pod-with-existing-rules
+  labels:
+    app: pod-with-existing-rules
+spec:
+      containers:
+      - name: webserver
+        image: buoyantio/iptables-tester:v1
+        env:
+         - name: PORT
+           value: "8080"
+        command: ["go", "run", "/go/test_service/test_service.go"]
+        ports:
+        - name: http
+          containerPort: 8080
+      - name: other-container
+        image: buoyantio/iptables-tester:v1
+        env:
+         - name: PORT
+           value: "9090"
+        command: ["go", "run", "/go/test_service/test_service.go"]
+        ports:
+        - name: http
+          containerPort: 9090
+      initContainers:
+      - name: iptables-test
+        image: ghcr.io/linkerd/proxy-init:latest
+        imagePullPolicy: Never
+        args: ["-p", "8080",  "-o", "8080", "-u", "2102"]
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            add:
+            - NET_ADMIN
+            - NET_RAW
+          privileged: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: false
+          runAsUser: 0
+        volumeMounts:
+        - mountPath: /run
+          name: linkerd-proxy-init-xtables-lock
+      - name: linkerd-init
+        image: ghcr.io/linkerd/proxy-init:latest
+        imagePullPolicy: Never
+        args: ["-p", "8080",  "-o", "8080", "-u", "2102"]
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            add:
+            - NET_ADMIN
+            - NET_RAW
+          privileged: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: false
+          runAsUser: 0
```

Comment:
These two containers—`iptables-test` and `linkerd-init`—are supposed to conflict? So `iptables-test` I assume is created first, and then `linkerd-init` is created second which results in the rules already existing? If so, where are we confirming that? I'd expect to see something in the tests that looks at the logs for this container and asserts that.

Reply:
Good point -- I can add that to the test to make sure we are asserting that `iptables-test` has already ran.